src/Microsoft.DotNet.Configurer/xlf/LocalizableStrings.zh-Hans.xlf

```diff
@@ -14,7 +14,7 @@ Use 'dotnet --help' to see available commands or visit: https://aka.ms/dotnet-cl
 
 遥测
 ---------
-.NET Core 工具收集用法数据，以便帮助改善用户体验。数据是匿名的，且不包括命令行参数。数据由 Microsoft 收集并与社区共享。可使用喜欢的 shell 将环境变量 DOTNET_CLI_TELEMETRY_OPTOUT 设置为 “1” 或 “true”，从而选择推出遥测。
+.NET Core 工具收集用法数据，以便帮助改善用户体验。数据是匿名的，且不包括命令行参数。数据由 Microsoft 收集并与社区共享。可使用喜欢的 shell 将环境变量 DOTNET_CLI_TELEMETRY_OPTOUT 设置为 “1” 或 “true”，从而选择退出遥测。
```

Comment:
Does the equivalent of this word still exist in the updated English text:

```
Welcome to .NET Core!
---------------------
Learn more about .NET Core: https://aka.ms/dotnet-docs
Use 'dotnet --help' to see available commands or visit: https://aka.ms/dotnet-cli-docs</source>
```

If not, there's no need to make this change as it will be undone regardless when the translation is updated to match the new source text.

Reply:
See https://github.com/dotnet/cli/commit/42f97b002b37488ae5f2a67925516d4cde249566#diff-c581cb12bafb946badc411b815839156 for the English text change

Reply:
Looks like this is moved to another string.